pom.xml

```diff
@@ -46,6 +46,11 @@
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Is this necessary? Also current tests are on JUnit 5. It looks like yours might be on JUnit 4.

Reply:
You're right it's not.